apps/nextjs/src/app/api/aila-download-all/route.ts

```diff
@@ -152,7 +152,7 @@ async function getHandler(req: Request): Promise<Response> {
 
         const { data } = res;
 
-        const filename = `${lessonTitle} - ${lessonExport.id} - ${getReadableExportType(
+        const filename = `${lessonTitle} - ${lessonExport.id.slice(0, 5)} - ${getReadableExportType(
```

Comment:
Have you sanity checked what the ids look like? I know some of the IDs around exports are UUIDs which can have the same prefix